util/base_test.py

```diff
@@ -17,3 +19,37 @@ def __init__(self, *args, **kwargs):
 
     def setUp(self):
         User.objects.create_user(**self._user)
+
+
+class BaseImageTest(BaseSingleUserTest):
+    """Base class for the image tests. """
```

Comment:
`Base class for the image tests that need to have an images created before running.`